src/FSFULLRWD.xml

```diff
@@ -0,0 +1,29 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<SPDXLicenseCollection xmlns="http://www.spdx.org/license">
+   <license isOsiApproved="false" licenseId="FSFULLRWD"
+   name="FSF Unlimited License (With License Retention
+   and Warranty Disclaimer)" listVersionAdded="3.19">
+      <crossRefs>
+         <crossRef>https://lists.gnu.org/archive/html/autoconf/2012-04/msg00061.html (first reference)</crossRef>
```

Comment:
Do we require the `<crossRef>` to be a URL? I'm assuming this is what the checker uses to see if the link is live... If so, then we might want to remove the `(first reference)` comment at the end of this.

Reply:
good point, not sure, but will remove!